src/components/organisms/Hero.jsx

```diff
@@ -137,7 +138,7 @@ function Hero() {
         <HeroHeading>
           <BaseTwoLogoText id="hero-logo" />
         </HeroHeading>
-        <HeroTagline> We turn ideas into beautiful software </HeroTagline>
+        <HeroTagline> Our mission is to solve problems that matter. We believe that best outcomes start on a solid foundation: open, honest, and equitable partnerships. Together we make exceptional the standard. </HeroTagline>
```

Comment:
🤔 Weird that there was previously leading/trailing space in this node. I _think_ that should be safe to remove.

```suggestion
        <HeroTagline>Our mission is to solve problems that matter. We believe that best outcomes start on a solid foundation: open, honest, and equitable partnerships. Together we make exceptional the standard.</HeroTagline>
```